pkg/blockatlas/marketdata.go

```diff
@@ -46,6 +46,7 @@ type Rate struct {
 	Currency  string  `json:"currency"`
 	Rate      float64 `json:"rate"`
 	Timestamp int64   `json:"timestamp"`
+	Provider  string  `json:"provider,omitempty"`
```

Comment:
Should we add name Provider?

Reply:
this parameter doesn't be returned by the API, only to save the rates in the database and keep the highest priority